docs/outlook/autolaunch.md

```diff
@@ -490,7 +490,14 @@ Some Office.js APIs that change or alter the UI aren't allowed from event-based
   - `displayDialogAsync`
   - `messageParent`
 
-### Requesting external data
+### Preview features in event handlers (Outlook on Windows)
+
+To test [Outlook features that are in preview](/javascript/api/requirement-sets/outlook/preview-requirement-set/outlook-requirement-set-preview) in event handlers on Outlook on Windows, you must configure your computer's registry as follows.
+
+1. In the registry, navigate to `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\Outlook\Options\WebExt\Developer`. If the key doesn't exist, create it.
+1. Create an entry named `EnableBetaAPIsInJavaScript` and set its value to `1`.
+
```

Comment:
consider including the outlook-beta-registry-key image we've used in the past (see [this PR](https://github.com/OfficeDev/office-js-docs-pr/pull/2690/files))